docs/reference/deploy/master-config-reference.rst

```diff
@@ -217,6 +217,20 @@ warning is returned. The default value is ``true``.
 
 Optional. Specify a human-readable name for this cluster.
 
+**********************
+ ``ui_customization``
+**********************
+
+Optional. Applies only to Determined Enterprise Edition. This section houses options to customize
+the UI.
+
+``logo_path``
+=============
+
+Path to variations of a user provided logo to be shown in the UI. This path should be available and
+reachable to the master and a valid image file. SVG format is recommended but other common web image
+formats are supported as well.
+
```

Comment:
``logo_path``
=============

Specifies the path to a user-provided logo to be shown in the UI. Ensure the path is accessible and reachable by the master service. The logo file should be a valid image format, with SVG recommended.